scripts/remote

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
I think this script should exit on error. Consider adding the line
```
set -e
```
to enable this.